.github/workflows/lint-app.yml

```
@@ -50,7 +50,19 @@ jobs:
               cd app
               yarn format:check
 
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)